Header header
float32[] range
float32[] bearing
bool meas_update

